hack/test-cnv.sh

```diff
@@ -31,4 +32,5 @@ ${TESTS_BINARY} \
     -ginkgo.succinct \
     -oc-path="$(which oc)" \
     -kubectl-path="$(which oc)" \
-    -utility-container-prefix=quay.io/kubevirt
+    -utility-container-prefix=quay.io/kubevirt \
+    -test.timeout=75m
```

Comment:
Is there a big enough margin so that we are not getting hit by that later on, i.e. when we decide to run more tests? Maybe rather use a value that let's us exit slightly before the test pod timeout so we at least have test results, but are not impacted when the tests take a little longer?

Reply:
The margin here is not too large. I'll increase to 120m which is more flexible and still remains within the job timeout.